### ISSUE TITLE
Bot displays error when executing any (?) command

### ISSUE THREAD

**Issue by SuperSlimeStar** [esmBot/esmBot#102]
The bot popped up with an error message whenever I try any command on it. There may be exceptions, but I don’t want to go through every command. Photo example below
![58B4ACD6-D097-4CAF-B077-10B3AB19580D](https://user-images.githubusercontent.com/83443662/116651867-1c2c2680-a952-11eb-913c-fcb81a210891.jpeg)

**Comment by TheEssem** [comment]
What's the actual content of the file?

**Comment by SuperSlimeStar** [comment]
I used a gif that worked previously. I also got it working in a server but it still refuses to work in a dm.

**Comment by samhza** [comment]
The bot fails on any command in a DM channel.
```
Message: TypeError: this.message.channel.createMessage is not a function

Stack Trace: TypeError: this.message.channel.createMessage is not a function
    at PingCommand.run (/home/esmBot/esmBot/commands/general/ping.js:5:52)
    at module.exports (/home/esmBot/esmBot/events/messageCreate.js:89:39)
    at runMicrotasks (<anonymous>)
```

**Comment by TheEssem** [comment]
Should be fixed with 56113a1, which will be part of the 1.5.2 release.
